app/misc/graphing.ts

```diff
@@ -336,7 +362,10 @@ function makeBasicNode(c, column: number) {
 
     if (flag) {
         id = basicNodeId++;
+        tagid = id + 1;
```

Comment:
Are these supposed to be unique? The line above adds 1, now you're adding another one. Perhaps a comment here?

Reply:
I'll comment something in, this may be rendered obsolete from the unique identifiers, but it might be still there for spacing purposes. I'll look into this, thanks!

Reply:
Id deals with branch array identification, and tagid with tag array identification. If we try to increment basicNodeId again, it fails. But we need tagid to be one ahead of branch id else the whole thing conflicts with itself. basicNodeId++ actually increments basicNodeId at the same time as storing it in id.

Reply:
Alright, I would suggest adding something to this effect here. Also, consider using ++?